examples/problem_decomposition/mifno.ipynb

```diff
@@ -338,62 +354,18 @@
-    "\n",
-    "Below, we treat a given fragment with Quantum Imaginary Time Evolution (QITE) to obtain its energy. We made sure to enter values for `n_electrons` and `n_spinorbitals` that are consistent with our MI-FNO parameters.\n",
-    "\n",
-    "This code cell can take several minutes to execute. After its execution, the value of `qite.final_energy` should be about `-15.5760`."
+    "Now that we are able to construct `FermionOperator` and `QubitOperator` objects representing the subproblems defined by our fragments, we are free to throw them at any [algorithm available](https://github.com/goodchemistryco/Tangelo/tree/main/tangelo/algorithms) in Tangelo. You could even decide to use your own custom workflow instead. Each fragment that is relevant to you could be solved with a different quantum solver, if you wish to do so."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 7,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Iteration 1 of QITE with starting energy -15.560098380997923\n",
-      "Iteration 2 of QITE with starting energy -15.565635973113471\n",
-      "Iteration 3 of QITE with starting energy -15.56926613851335\n",
-      "Iteration 4 of QITE with starting energy -15.571640808284947\n",
-      "Iteration 5 of QITE with starting energy -15.573192508327143\n",
-      "Iteration 6 of QITE with starting energy -15.574206095376253\n",
-      "Iteration 7 of QITE with starting energy -15.574868306596551\n",
-      "Iteration 8 of QITE with starting energy -15.575301213500483\n",
-      "Iteration 9 of QITE with starting energy -15.575584484500121\n",
-      "Iteration 10 of QITE with starting energy -15.575770073238283\n",
-      "Iteration 11 of QITE with starting energy -15.575891850852267\n",
-      "Final energy of QITE is -15.575971903346913\n"
-     ]
-    },
-    {
-     "data": {
-      "text/plain": [
-       "-15.575971903346913"
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "from tangelo.algorithms.projective import QITESolver\n",
-    "from tangelo.toolboxes.operators import qubitop_to_qubitham\n",
-    "\n",
-    "qubit_ham = qubitop_to_qubitham(qu_op, mapping=\"scBK\", up_then_down=True)\n",
-    "qite = QITESolver({\n",
-    "    \"qubit_hamiltonian\": qubit_ham, \n",
-    "    \"qubit_mapping\":\"scBK\", \n",
-    "    \"n_electrons\": n_electrons, \n",
-    "    \"n_spinorbitals\": n_spinorbitals,\n",
-    "    \"up_then_down\": True,\n",
-    "    \"min_de\": 1e-4\n",
-    "})\n",
-    "qite.build()\n",
-    "qite.simulate()"
+    "from openfermion.linalg import eigenspectrum\n",
+    "\n",
+    "minimal_eigenvalue = eigenspectrum(qu_op).min()"
```

Comment:
curious where the energy difference comes from btw, i expected this line to return something extremely accurate since we are diagonalizing the fragment.

Reply:
The classical solver for MIFNO is HBCI, so it could be different to the FCI solution (should it be?).